test/unit/stPacket/validation/findDuplicateDPObjectsByIndices.js

```diff
@@ -0,0 +1,31 @@
+const findDuplicateDPObjectsByIndices = require('../../../../lib/stPacket/validation/findDuplicateDPObjectsByIndices');
+
+const getDPContractFixture = require('../../../../lib/test/fixtures/getDPContractFixture');
+const getDPObjectsFixture = require('../../../../lib/test/fixtures/getDPObjectsFixture');
+
+describe('findDuplicateDPObjectsByIndices', () => {
+  let rawDPObjects;
+  let dpContract;
+
+  beforeEach(() => {
+    rawDPObjects = getDPObjectsFixture().map(o => o.toJSON());
+    dpContract = getDPContractFixture();
+  });
+
```

Comment:
Please make sure that you have object types without indices, with non-unique indices, with only one object and so on according to the function algorithm.